cmsBuild

```diff
@@ -619,6 +621,31 @@ def downloadPip(source, dest, options):
                     i = i + 1
                 else:
                     pip_opts = pip_opts + ' ' + spSrc[i]
+                    if "no-binary" in spSrc[i] and "all" not in spSrc[i]:
+                        isSourceDownload=False #not totally robust - but basically use pip if source is overridden
+
+    if isSourceDownload:
+        log("Looking for sources at https://pypi.org/pypi/"+pack.split('=')[0]+"/json")
+        fj=urlopen("https://pypi.org/pypi/"+pack.split('=')[0]+"/json")
+        data=json.load(fj)
+        url=None
+        if "releases" in data and pack.split('=')[2] in data["releases"]:
+            for file in data["releases"][pack.split('=')[2]]:
+                if file["packagetype"] == "sdist":
+                    url=file["url"]
+        if url is not None:
```

Comment:
```suggestion
        if url:
```

Reply:
it seems is not None is the more standard formulation.. "Comparisons to singletons like None should always be done with is or is not, never the equality operators."